src/components/Supporters/index.jsx

```diff
@@ -0,0 +1,227 @@
+import React from "react";
+import styles from "./styles.module.css";
+import Link from "@docusaurus/Link";
+import ThemedImage from "@theme/ThemedImage";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+/* data of supporters */
+const financial = [
+    {
+        name: "Nvidia",
+        link: "https://www.nvidia.com/",
+        light: "img/supporters/nvidia_light.svg",
+        dark: "img/supporters/nvidia_dark.svg",
+        width: 250,

```

Comment:
What are the units here?

Reply:
> What are the units here?

It is `px` unit. But I can't use the word `px` because it expects an integer value.